tasks/main.yml

```diff
@@ -6,7 +6,9 @@
     appends: true
   run_once: true
   delegate_to: localhost
-  when: groups["all"]|select('match', '^.+\.lxc$')|list|count
+  when:
+    - project_docker_network is defined
```

Comment:
`project_docker_network` should be replaced by `boot_docker_network` in this file too.